src/Deals.php

```diff
@@ -26,6 +27,17 @@ public function all($limit = false, $offset = 0)
 		return $this->formatDealsForReturn($deals);
 	}
 
+	public function getCountByCategorySlug($slug, $limit = false, $offset = 0)
+        {
+                $query = $this->getBaseQuery($limit, $offset);
```

Comment:
Is there weird spacing in your sublime here? Or is this just on github. Are you using 4 spaces for tabs? It just looks a little sloppy so if we could clean it up that would be good.